includes/Emails/VendorNewOrder.php

```diff
@@ -0,0 +1,194 @@
+        $this->title          = __( 'Dokan Vendor New Order', 'dokan-lite' );
+        $this->description    = __( 'New order emails are sent to chosen recipient(s) when a new order is received.', 'dokan-lite' );
+        $this->template_html  = 'emails/vendor-new-order.php';
+        $this->template_plain = 'emails/plain/vendor-new-order.php';
+        $this->template_base  = DOKAN_DIR.'/templates/';
+        $this->placeholders   = array(
+            '{site_title}'   => $this->get_blogname(),
+            '{order_date}'   => '',
+            '{order_number}' => '',
+        );
+
+        // Triggers for this email.
+        add_action( 'woocommerce_order_status_pending_to_processing_notification', array( $this, 'trigger' ), 10, 2 );
+        add_action( 'woocommerce_order_status_pending_to_completed_notification', array( $this, 'trigger' ), 10, 2 );
+        add_action( 'woocommerce_order_status_pending_to_on-hold_notification', array( $this, 'trigger' ), 10, 2 );
+        add_action( 'woocommerce_order_status_failed_to_processing_notification', array( $this, 'trigger' ), 10, 2 );
+        add_action( 'woocommerce_order_status_failed_to_completed_notification', array( $this, 'trigger' ), 10, 2 );
+        add_action( 'woocommerce_order_status_failed_to_on-hold_notification', array( $this, 'trigger' ), 10, 2 );
+        
+        // Call parent constructor.
+        parent::__construct();
+        
+        // Other settings.
+        $this->recipient = 'vendor@ofthe.product';
+    }
+
+    /**
+     * Get email subject.
+     *
+     * @since  3.1.0
+     * @return string
+     */
+    public function get_default_subject() {
+        return __( '[{site_title}] New customer order ({order_number}) - {order_date}', 'dokan-lite' );
+    }
+
+    /**
+     * Get email heading.
+     *
+     * @since  3.1.0
+     * @return string
+     */
+    public function get_default_heading() {
+        return __( 'New Customer Order: #{order_number}', 'dokan-lite' );
+    }
+
+    /**
+     * Trigger the sending of this email.
+     *
+     * @param int $order_id The Order ID.
+     * @param array $order.
+     */
+    public function trigger( $order_id, $order = false ) {
+
+        if ( ! $this->is_enabled() ) {
+            return;
+        }
+
+        $this->setup_locale();
+        if ( $order_id && ! is_a( $order, 'WC_Order' ) ) {
+            $order = wc_get_order( $order_id );
+        }
+
+        if ( is_a( $order, 'WC_Order' ) ) {
+            $this->object                         = $order;
+            $this->placeholders['{order_date}']   = wc_format_datetime( $this->object->get_date_created() );
+            $this->placeholders['{order_number}'] = $this->object->get_order_number();
+        }
+        
+        $sellers = dokan_get_seller_id_by_order( $order_id );
+        if ( empty( $sellers ) ) {
+            return;
+        }
+        
+        // check has sub order 
+        if ( dokan_is_sub_order( $order_id ) ) {
```

Comment:
If it's a sub-order then why we're looping through all the sellers? Shouldn't be when it's not a sub-order?